common/src/test/java/tachyon/util/network/NetworkAddressUtilsTest.java

```
@@ -162,7 +162,25 @@ public void getBindAddress(ServiceType service) throws Exception {
         workerAddress);
 
     // connect host and wildcard bind host with port
-    conf.set(Constants.WORKER_RPC_PORT, "20000");
+    switch (service) {
+      case MASTER_RPC:
+        conf.set(Constants.MASTER_RPC_PORT, "20000");
+        break;
+      case MASTER_WEB:
+        conf.set(Constants.MASTER_WEB_PORT, "20000");
+        break;
+      case WORKER_RPC:
+        conf.set(Constants.WORKER_RPC_PORT, "20000");
+        break;
+      case WORKER_DATA:
+        conf.set(Constants.WORKER_DATA_PORT, "20000");
+        break;
+      case WORKER_WEB:
+        conf.set(Constants.WORKER_WEB_PORT, "20000");
+        break;
+      default:
+        break;
```

The default case should probably throw an exception